tb_plugin/torch_tb_profiler/profiler/run_generator.py

```diff
@@ -22,6 +22,7 @@ def __init__(self, worker, span, profile_data: RunProfileData):
 
     def generate_run_profile(self):
         profile_run = RunProfile(self.worker, self.span)
+        profile_run.framework = self.profile_data.framework
```

Comment:
is it better to use bool is_pytorch_lightning than the "framework"?

Reply:
Will we have other framework in future?

Reply:
the "framework" is specific to PyTorch Lightning. And we cannot guarantee all other framework be tagged with "Framework". As the result, keep the implementation private by exposing more meaningful name like "is_pytroch_lighting" is much better.

Reply:
Done